src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceDeploy.kt

```diff
@@ -10,9 +10,9 @@ import java.io.File
 open class InstanceDeploy : Instance() {
 
     @Internal
-    val file = aem.obj.file {
-        common.prop.string("instance.deploy.url")?.let { url ->
-            fileProvider(aem.obj.provider { common.resolveFile(url) })
+    val packages = aem.obj.files() {
+        common.prop.list("instance.deploy.url")?.let { urls ->
```

Comment:
is it backward compatible? how `common.prop.list` behaves when there is single url specified?